pv211_utils/cranfield/eval.py

```diff
@@ -27,9 +29,9 @@ class CranfieldEvaluation(EvaluationBase):
 
     """
     def __init__(self, system: CranfieldIRSystemBase, judgements: Set[CranfieldJudgementBase],
-                 leaderboard: Optional[CranfieldLeaderboard] = None,
+                 leaderboard: Optional[CranfieldLeaderboard] = None, k: Optional[int] = 1400,
```

Comment:
Please, unit-test in `test/test_evaluation_metrics.py` that specifying any `k` larger than the size of the dataset (here we specify 1400 but Cranfield only contains 1398 articles) does not affect the evaluation scores. It clearly works as intended now, but will be easily broken by further development.

Reply:
todo

Reply:
should be done